requirements.txt

```diff
@@ -1,16 +1,24 @@
 pathlib
-typing
-tqdm
-pandarallel
-haversine
-scipy
-numpy
 datetime
-geopandas
+diffprivlib==0.5.0
```

Comment:
we should look into how to best declare dependencies. Might be better to do more >= than too strict version numbers